static/usage.html

```diff
@@ -1123,6 +1123,19 @@ <h3><a href="#sandboxed-google-play-esim">Privileged eSIM management</a></h3>
                     "Checking network info..." stage despite having a stable Internet connection,
                     you may need to call the USSD code <code>*#*#4636#*#*</code> and then enable
                     DSDS in the menu that is presented.</p>
+
+                    <!-- https://github.com/GrapheneOS/os-issue-tracker/issues/2162 -->
+                    <p><strong>Warning:</strong> Do not enable a PIN for an eSIM without first
+                    memorizing the eSIM's
+                    <a href="https://en.wikipedia.org/wiki/Personal_unblocking_key">PUK</a>.
+                    If you do not know your eSIM's PUK, you may be forced to factory-reset
+                    your device (<strong>thus wiping your data</strong>) to reinstall the
+                    stock OS, as follows. GrapheneOS automatically disables the proprietary
```

Comment:
```suggestion
                    stock OS, as follows. GrapheneOS automatically disables the proprietary
```
What does "as follows" refer to here?

Reply:
"As follows" refers to the remainder of the paragraph, i.e., the steps by which a user might end up unable to unlock the device:
* Device reboot triggers automatic disabling of Google's eSIM code
* Multiple PIN entries fail, causing the PIN to lock itself
* The "Disable eSIM" button on the SIM-unlock screen won't work
* So only the PUK will do.

It would be possible to turn the multiple sentences into a formatted list, but that seemed like it might be overkill -- especially since it seems the desired situation would be to at some point fix the "Disable eSIM" button, or at least to replace it with an "Ignore eSIM" button, or to have native eSIM management code.  Overall it seemed as if the eventual fate of this warning would be deletion.